src/orquestra/sdk/_base/_api/_wf_run.py

```diff
@@ -513,6 +516,54 @@ def list_workflow_runs(
     return runs
 
 
+def list_workspaces(
+    config: t.Union[ConfigName, "RuntimeConfig"],
+) -> t.Sequence[WorkspaceDef]:
+    """Get the list of all workspaces available to a user.
+    Warning: works only on CE runtimes
+
+    Args:
+        config: The name of the configuration to use.
+
+    Raises:
+        ConfigNameNotFoundError: when the named config is not found in the file.
+    """
+    # Resolve config
+    resolved_config = _resolve_config(config)
+
+    runtime = resolved_config._get_runtime()
+
+    return runtime.list_workspaces()
+
+
+def list_projects(
+    config: t.Union[ConfigName, "RuntimeConfig"],
+    workspace_id: t.Union[WorkspaceId, WorkspaceDef],
+) -> t.Sequence[ProjectDef]:
+    """Get the list of all workspaces available to a user.
+    Warning: works only on CE runtimes
+
+    Args:
+        config: The name of the configuration to use.
+        workspace_id: ID of the workspace to use
+    Raises:
+        ConfigNameNotFoundError: when the named config is not found in the file.
+    """
+    # Resolve config
+    resolved_config = _resolve_config(config)
+    workspace_id_resolved: str
+
+    workspace_id_resolved = (
+        workspace_id.workspace_id
+        if isinstance(workspace_id, WorkspaceDef)
+        else workspace_id
+    )
+
+    runtime = resolved_config._get_runtime()
+
+    return runtime.list_projects(workspace_id_resolved)
```

Comment:
Non-blocking suggestion: How would you feel about `resolved_workspace_id` for consistency with `resolved_config`?

```suggestion
    resolved_workspace_id: str

    resolved_workspace_id = (
        workspace_id.workspace_id
        if isinstance(workspace_id, WorkspaceDef)
        else workspace_id
    )

    runtime = resolved_config._get_runtime()

    return runtime.list_projects(resolved_workspace_id)
```